### PR TITLE
Enable PoolBlockingPeriod for .NET Standard + expose GetColumnSchema

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Contains below changes:

1. Below API was not exposed in .NET Standard DLLs, even though functionality was available.
   - SqlDataReader.GetColumnSchema()

2. Pool Blocking Period has no API limitations in .NET Standard 2.0 and 2.1, tested using "-p:ReferenceType=NetStandard", hence enabling it here. Would appreciate feedback/concerns on this change.

cc @saurabh500 @David-Engel

### CODE REVIEW

**Review by David-Engel — COMMENTED** [review]
Approved from a functional perspective. I don't know why it was originally excluded from netstandard.

**Review by johnnypham — APPROVED** [review]

**Review by JRahnama — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs [hunk 1]
```diff
@@ -5,35 +5,6 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs:5]: Does it require passing the [API review process](http://aka.ms/api-review)?

**Review by DavoudEshtehari — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs (hunk 1 above)
Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs:5]: Well, that's legacy link came from dotnet/corefx, not applicable to this library anymore.

Comment (DavoudEshtehari) [reply src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs:5]: So, It's good if we remove it.

**Pull request merged**